frontend/pages/collections/[slug].page.tsx

```diff
@@ -78,3 +103,113 @@ export const getStaticPaths: GetStaticPaths = async () => {
     fallback: "blocking",
   };
 };
+
+const MobileViewComponent: FC<CollectionProps> = ({ collection }: CollectionProps) => {
```

Comment:
These component names are a little generic. Can we use more descriptive names? Maybe `MobileSharingButtonGroup` or `MobileSharingButtons`?